lib/tasks/addon-install.js

```diff
@@ -31,33 +32,37 @@ module.exports = Task.extend({
       testing:    this.testing
     });
 
-    ui.startProgress(chalk.green('Installing addon package'), chalk.green('.'));
+    var packageInflection = inflector.inflect('package', packageNames.length);
```

Comment:
do we need a full inflection library for this? Seems like this inflection is trivial no?